bmHotbarClass.lua

```diff
@@ -548,7 +564,9 @@ function BMHotbarClass:RenderTabContextMenu()
             table.sort(charList, function(a, b) return a.key < b.key end)
             for _, value in ipairs(charList) do
                 if ImGui.MenuItem(value.displayName) then
-                    CopyLocalSet(value.key)
+                    -- CopyLocalSet(value.key)

```

Comment:
let's not add comments back into the branch.

Also is there a reason this was moved to the other coroutine? I am not against it but just curious why this was done.